pages/mindful.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Guide from "@/components/Guide/Guide";
+import NavButton from "@/components/NavButton/NavButton";
+import { useRouter } from "next/router";
+import GuidedMeditation from "@/components/GuidedMeditation/GuidedMeditation";
+
+export default function Entries() {
+  const router = useRouter();
+
+  return (
+    <>
+      <Guide text={"Guided Meditation"} />
+      <GuidedMeditation />
+
+      <NavButton handleClick={() => router.back}>back</NavButton>
```

Comment:
If the only purpose of the button is to navigat, a Link component woould be more appropriate

Reply:
We have this standardized NavButton we use everywhere. It leads back to the activites page.